Beta_Integrations/GmailSingleUser/GmailSingleUser.py

```diff
@@ -0,0 +1,904 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+import re
+import json
+import base64
+from datetime import datetime, timedelta
+import httplib2
+import sys
+from html.parser import HTMLParser
+from html.entities import name2codepoint
+from email.mime.audio import MIMEAudio
+from email.mime.base import MIMEBase
+from email.mime.image import MIMEImage
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from email.header import Header
+import mimetypes
+import random
+import string
+from apiclient import discovery
+from oauth2client.client import AccessTokenCredentials
+import itertools as it
+from cryptography.hazmat.primitives.ciphers.aead import AESGCM
+import urllib.parse
+
+''' GLOBAL VARS '''
+params = demisto.params()
+EMAIL = params.get('email', '')
+PROXY = params.get('proxy')
+DISABLE_SSL = params.get('insecure', False)
+FETCH_TIME = params.get('fetch_time', '1 days')
+OPROXY_URL = 'https://us-central1-oproxy-dev.cloudfunctions.net'  # disable-secrets-detection
```

Comment:
Fix to proper prod url. Use same method as we do with graph to configure custom url via auth_id and `@`. See: https://github.com/demisto/content/blob/master/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py#L26